tools/setup/lang.json

```diff
@@ -0,0 +1 @@
+{"mql4": 9, "matlab": 33, "vbnet": 45, "fsharp": 19, "tex": 40, "go": 34, "vb.net": 45, "julia": 4, "perl": 40, "objectpascal": 42, "mql": 9, "pascal": 42, "csharp": 47, "erlang": 10, "python": 46, "lisp": 18, "scheme": 14, "rb": 39, "math": 50, "bash": 7, "r": 37, "sas": 30, "quote": 50, "abap": 27, "c++": 48, "delphi": 42, "lua": 22, "javascript": 51, "js": 43, "swift": 41, "c": 49, "java": 50, "pl": 40, "c#": 47, "ruby": 39, "prolog": 16, "cobol": 26, "php": 44, "ada": 25, "dart": 28, "d": 29, "scala": 21, "awk": 1, "html": 30, "haskell": 15, "latex": 40, "objectivec": 35, "groovy": 13, "rust": 8, "objective-c": 35, "sql": 32, "mask": 2, "cpp": 48, "css": 48, "xml": 1}
```

Comment:
Can you pretty-print this with one language on each line?  And maybe sort it, either by value or by language name?  Since we may be editing this by hand, we don't really want it to have a single super-long line, since diffs will be much less readable that way :)